babel/util.py

```diff
@@ -134,6 +134,16 @@ def parse_future_flags(fp: IO[bytes], encoding: str = 'latin-1') -> int:
     return flags
 
 
+def remove_trailing_zeros_after_decimal(string: str, decimal_symbol: str) -> str:
```

Comment:
I don't think this should be a public-smelling API – maybe move it to `babel/numbers.py` and rename it to `_remove_trailing_zeros_after_decimal`?

Also, a docstring would be great in any case :)

Reply:
That's true. The function has been moved to `babel/numbers.py`, renamed to `_remove_trailing_zeros_after_decimal`, and now includes a docstring.